### PR TITLE
feat:Add support for group in the nav configuration of the press theme

### PR DESCRIPTION
**Pull request by jiaxinPiggy**
### Linked Issues
https://github.com/YunYouJun/valaxy/issues/477

### CODE REVIEW

File: packages/valaxy-theme-press/types/index.d.ts [hunk 1]
```diff
@@ -103,9 +103,16 @@ export interface NavItemLink {
   active?: string
 }
 
+export interface NavItemChildren {
```
Comment (YunYouJun) [inline packages/valaxy-theme-press/types/index.d.ts:106]: How about we align with the type of vitepress as much as possible?

https://github.com/vuejs/vitepress/commit/e2d4edf45b5ec890c088d3b0517b21a7b3eab9df

**Review by YunYouJun — CHANGES_REQUESTED** [review]
Can you just provide an example and screenshot preview?

For the example, I think we can apply it directly to the current document.

**Comment by jiaxinPiggy** [comment]
Sorry, I don't know how to provide an example. I can provide some screenshots
case1
case2
case3

**Comment by YunYouJun** [comment]
Thanks!

**Review by YunYouJun — APPROVED** [review]

**Pull request merged**